### PR TITLE
Assorted security fixes needed to open source the SDK

### PR DESCRIPTION
**Pull request by tmeyer2115**
- Removing templateUrl as an option from the SDK.
- Updating XSS vulnerabilities across multiple HBS templates.

### CODE REVIEW

File: src/core/constants.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /** @module */
 
 /** The current lib version, reported with errors and analytics */
-export const LIB_VERSION = 'v0.13';
+export const LIB_VERSION = 'v0.12';
```
Comment (alexisgrow) [inline src/core/constants.js:4]: Is this change intended?

Comment (tmeyer2115) [reply src/core/constants.js:4]: Yep, updating the LIB_VERSION should be the last commit of the release. Otherwise, we will be attempting to fetch the default set of compiled templates from a location that doesn't exist in the CDN: https://assets.sitescdn.net/answers/v0.13/answerstemplates.compiled.min.js

Comment (alexisgrow) [reply src/core/constants.js:4]: Ahh, gotcha, that makes sense!

**Review by alexisgrow — APPROVED** [review]
